apps/web/src/hooks/useGetBlockCountdown.ts

```diff
@@ -1,6 +1,7 @@
 import { useEffect, useRef, useState } from 'react'
 import { BSC_BLOCK_TIME } from 'config'
-import { bscRpcProvider } from 'utils/providers'
+import { publicClient } from 'utils/wagmi'
+import { ChainId } from '@pancakeswap/sdk'
```

Comment:
we've moved the `ChainId` to a separate package called `@pancakeswap/chains`. Let's import from the new pkg